docs/styles/styles.css

```diff
@@ -77,6 +77,7 @@
 @layer layout {
   :root {
     --uxdot-masthead-max-height: 72px;
+    --uxdot-masthead-z-index: 4;
```

Comment:
I actually think this is a bug in the health index instead of the site styles.  IMHO health-index shouldn't use a such a high z-index of 10, when 1 is sufficient.  /cc @markcaron  Going to split this off into a separate issue.